bitwatch/umbrel-app.yml

```
@@ -29,17 +29,11 @@ gallery:
   - 2.jpg
   - 3.jpg
 releaseNotes: >-
-  This release completely rebuilds the websocket and API request queue logic and includes bug, performance, and UX fixes.
-
-    - new method for mempool.space websocket subscription management
-      - now managed per address to prevent hundreds of websocket subscriptions
-      - improved websocket logic for adding/editing/deleting addresses
-      - showing websocket subscription status in address row
-    - clearer UI for zero balances (opacity reduction)
-    - tighter UI (reduced margin/padding)
-    - fix auto-acceptance bug (failing to alert telegram when alert is set in some cases)
-    - descriptor and extended key rows now show balance summary of address lists
-    - more robust e2e tests
+  Telegram healthcheck, Performance improvements, and remembers configs:
+
+  - saves bitcoin/satoshi viewing preference to local storage
+  - adds a healthcheck for telegram connection to prevent loss of telegram notifications
+  - removes excess UI console logs that were causing UI performance issues
```

Please adapt the release notes formatting.
```suggestion
  Telegram healthcheck, Performance improvements, and remembers configs:
    - saves bitcoin/satoshi viewing preference to local storage
    - adds a healthcheck for telegram connection to prevent loss of telegram notifications
    - removes excess UI console logs that were causing UI performance issues
```